src/routes/productRouter.ts

```diff
@@ -106,35 +108,10 @@ router.get(
   productController.sellerGetProduct
 );
 
-router.post(
-  "/buyer-add-product-wishList/:id",
-  userAuthorization(["buyer"]),
-  isProductExistToWishlist,
-  productController.buyerAddProductToWishList
-);
-router.get(
-  "/buyer-view-whishlist-product",
-  userAuthorization(["buyer"]),
-  isUserWishlistExist,
-  productController.buyerViewWishLists
-);
-router.get(
-  "/buyer-view-whishlist-product/:id",
-  userAuthorization(["buyer"]),
-  isUserWishlistExistById,
-  productController.buyerViewWishList
-);
-router.delete(
-  "/delete-whishlist-products",
-  userAuthorization(["buyer"]),
-  isUserWishlistExist,
-  productController.buyerDeleteAllProductFromWishlist
-);
-router.delete(
-  "/delete-whishlist-product/:id",
-  userAuthorization(["buyer"]),
-  isUserWishlistExistById,
-  productController.buyerDeleteProductFromWishList
-);
+router.post("/buyer-add-product-wishList/:id",userAuthorization(["buyer"]),isWishListExist,isWishListProductExist,productController.buyerAddProductToWishList)
+router.get("/buyer-view-wishlist-products",userAuthorization(["buyer"]),isUserWishlistExist,productController.buyerViewWishLists)
```

Comment:
change this from **`productController.buyerViewWishLists`** to **`productController.buyerViewWishListProducts`**